packages/scandipwa/src/route/Checkout/Checkout.component.tsx

```diff
@@ -335,6 +337,52 @@ export class CheckoutComponent extends PureComponent<CheckoutComponentProps> {
         );
     }
 
+    renderItemsForSummaryFallback(count: number): ReactElement {
+        const maxCount = 3;
```

Comment:
probably better to move it to the `.config` file and rename it with a more descriptive name.